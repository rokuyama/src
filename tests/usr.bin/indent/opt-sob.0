/* $NetBSD: opt-sob.0,v 1.3 2021/10/11 16:30:54 rillig Exp $ */
/* $FreeBSD$ */

/* $ FIXME: There are lots of 'optional blank lines' here that should */
/* $ FIXME: be swallowed. */
void		function_declaration(void);


int
function_with_0_blank_lines(void)
{
    int		var;
    var = value;
    return var;
}

int
function_with_1_blank_line(void)
{

    int		var;

    var = value;

    return var;

}


int
function_with_2_blank_lines(void)
{


	int		var;


	var = value;


	return var;


}
