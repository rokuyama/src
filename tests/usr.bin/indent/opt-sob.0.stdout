/* $NetBSD: opt-sob.0.stdout,v 1.3 2021/10/11 16:30:54 rillig Exp $ */
/* $FreeBSD$ */

void		function_declaration(void);


int
function_with_0_blank_lines(void)
{
	int		var;
	var = value;
	return var;
}

int
function_with_1_blank_line(void)
{

	int		var;

	var = value;

	return var;

}


int
function_with_2_blank_lines(void)
{


	int		var;


	var = value;


	return var;


}
